src/main/java/sim/coffee/OrderList.java

```diff
@@ -89,6 +94,29 @@ public BigDecimal getTotalIncome() {
 		return sum;
 	}
 
+	/**
+	 * Income method for the day, 
+	 * @return cummulative income for orders received for a specific day
+	 */
+	public BigDecimal getTodayIncome(LocalDate date) {
```

Comment:
Method should probably be renamed to reflect that it can be any day (e.g. `getDayIncome`). Tip: You can use F2 in VSCode to quickly rename any method or variable (will updated all uses of it too).